packages/wrangler/e2e/provision.test.ts

```diff
@@ -133,12 +131,10 @@ describe("provisioning", { timeout: TIMEOUT }, () => {
 		expect(normalize(output)).toMatchInlineSnapshot(`
 			"Total Upload: xx KiB / gzip: xx KiB
 			Your Worker has access to the following bindings:
-			- KV Namespaces:
-			  - KV
-			- D1 Databases:
-			  - D1
-			- R2 Buckets:
-			  - R2
+			Binding                 Resource
+			env.KV (inherited)      KV Namespace
+			env.D1 (inherited)      D1 Database
+			env.R2 (inherited)      R2 Bucket
```

Comment:
This was not the main point of this PR, but I thought I might as well clean up some of the provisioning logging while I was at it